content/en/docs/overview/singleton-sdk/Angular Client Introduction/translate-directive.md

```diff
@@ -28,16 +28,26 @@ class L10nDirective implements AfterViewInit {
 
 #### **Inputs**
 
-| Attribute |  Type   | Required | Description                                                  |
-| :-------: | :-----: | :------: | :------------------------------------------------------------: |
-|    key    | String  | Required | Bound to L10nDirective.l10n.<br/>Define the key to identify the translation, it should name like this: component_module_page_control_shortmsg. e.g. web_settings_stats_statsTable_host; |
+| Attribute |  Type   | Required | <div style="text-align:center">Description</div>                                                                                                                                                                                                |
+|:---------:|:-------:|:--------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    key    | String  | Required | Bound to L10nDirective.l10n.<br/>Define the key to identify the translation, it should name like this: component_module_page_control_shortmsg. e.g. web_settings_stats_statsTable_host;                                                         |
 |  source   | String  |    No    | Bound to L10nDirective.source.<br/>English string as default value, API will return it when there's no translation found either from cache or remote singleton server.<br/>If source is null, will looking for source string from sourceBundle. |
-|  params   | Array[] |    No    | Bound to L10nDirective.params.<br/>Parameter array, it's used to format the translation/source when they contain placeholders. |
+|  params   | Array[] |    No    | Bound to L10nDirective.params.<br/>Parameter array, it's used to format the translation/source when they contain placeholders.                                                                                                                  |
 
 
 #### **Example Code**
 
-![translate-directive-1](https://github.com/zmengjiao/singleton/raw/website/content/en/images/translate-directive/translate-directive-1.png)
+```
+// Translate directive with source from sourceBundle, so there is no source in template
+<span l10n='VIP.description' [params]="['VIP for Angular client']" ></span>
```

Comment:
<span l10n='singleton.description' [params]="['Singleton for Angular client']" ></span>

Reply:
Done!